CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[BREAKING CHANGE]** Use relative imports so applications can resolve types.
```

Comment:
Is it really breaking for the package consumers?

Reply:
I just updated the description to explain why 👌

> **UPDATE**: The change is marked as breaking because unresolved types are no longer resolved as `any` and can break usages.